job.go

```
@@ -227,15 +233,29 @@ func parseToJob[T, R any](data []byte) (iJob[T, R], error) {
 
-// Close closes the job and its associated channels.
+// close closes the job and its associated channels.
 // the job regardless of its current state, except when locked.
-func (j *job[T, R]) Close() error {
+func (j *job[T, R]) close() error {
 	if err := j.isCloseable(); err != nil {
 		return err
 	}
 
 	j.resultChannel.Close()
+	j.Ack()
```

golangci-lint (1.64.8)

Error return value of `j.Ack` is not checked

(errcheck)